components/hubspot/actions/common-create-object.mjs

```diff
@@ -36,6 +36,15 @@ export default {
     } = this;
     const objectType = this.getObjectType();
 
+    // checkbox (string[]) props must be semicolon separated strings
+    Object.keys(properties)
```

Comment:
The reason I don't like to use `Object.entries()` is because it's considerably slower than `Object.keys()` and `Object.values()`. In our case it probably won't make much of a difference, but...